Marlin/Configuration.h

```diff
@@ -258,6 +258,22 @@
   #define SWITCHING_TOOLHEAD_X_POS        { 215, 0 }  // (mm) X positions for parking the extruders
 #endif
 
+/**
+ * Magnetic Switching Toolhead
+ *
+ * Support for swappable and dockable toolheads
+ * via magnetic docking mechanism using movement and no serveo.
+ */
+//#define MAGNETIC_SWITCHING_TOOLHEAD
+#if ENABLED(MAGNETIC_SWITCHING_TOOLHEAD)
+  #define MAGNETIC_SWITCHING_TOOLHEAD_Y_POS         -49         // (mm) Y position of the toolhead dock
+  #define MAGNETIC_SWITCHING_TOOLHEAD_Y_RELEASE       5         // (mm) Security distance Y axis
+  #define MAGNETIC_SWITCHING_TOOLHEAD_Y_SECURITY     43         // (mm) Security distance Y axis
+  #define MAGNETIC_SWITCHING_TOOLHEAD_Y_CLEAR        48         // (mm) Minimum distance from dock for unobstructed X axis
+  #define MAGNETIC_SWITCHING_TOOLHEAD_X_SECURITY    -35         // (mm) Security distance X axis
+  #define MAGNETIC_SWITCHING_TOOLHEAD_X_POS        { 115, 215 }  // (mm) X positions for parking the extruders
+#endif
```

Comment:
If this feature isn't meant to be used in conjunction with `SWITCHING_TOOLHEAD` (and I presume for now it is not) then it would be better to have just one shared setting for `X_POS`, `Y_POS`, `Y_SECURITY`, `Y_CLEAR`, and so on.

Reply:
Thanks for your suggestion.
Name space of settings has shared with  SWITCHING_TOOLHEAD.

Reply:
Looks good, thanks!